src/cborparser.c

```diff
@@ -108,7 +108,7 @@ static CborError preparse_value(CborValue *it)
     }
 
     size_t bytesNeeded = descriptor < Value8Bit ? 0 : (1 << (descriptor - Value8Bit));
```

Comment:
Is the `1` being left-shifted going to be interpreted as an `int` or as a `size_t`?